client/src/components/LoginPage/index.js

```diff
@@ -1,86 +1,110 @@
 import React, { useState } from "react";
-import { Button, makeStyles } from "@material-ui/core";
-import TextField from "@material-ui/core/TextField";
+import { Button } from "@material-ui/core";
+import UserContext from "../../UserContext";
 import Footer from "./Footer";
+import InputField from "./InputField";
 import "./Login.css";
+import Dialog from "../shared/Dialog";
+import axios from "axios";
 
-const styles = makeStyles({
-  textBox: {
-    margin: 10,
-    width: "80%",
-  },
-});
-
-// const [state name, function to update state]
-// body = default state
 const LoginPage = (props) => {
-  const [user, setTempUser] = useState({
+  const { user, setUser } = React.useContext(UserContext);
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
+  const [error, setError] = useState(false);
+  const [temp, setTempUser] = useState({
     username: "",
     email: "",
     password: "",
   });
 
   // called when user types into fields
-  const handleChange = (e) => {
+  // if any fields are red (error), remove them at this time
+  // update temp user fields based on which field was typed into
+  const handleChange = (e, temp) => {
     const { name, value } = e.target;
-    if (name === "username") {
-      setTempUser({
-        username: value,
-        email: user["email"],
-        password: user["password"],
-      });
-    } else if (name === "email") {
-      setTempUser({
-        username: user["username"],
-        email: value,
-        password: user["password"],
-      });
+    if (name === "email") {
+      setTempUser({ ...temp, email: value });
+      setEmailError(false);
     } else if (name === "password") {
-      setTempUser({
-        username: user["username"],
-        email: user["email"],
-        password: value,
-      });
+      setTempUser({ ...temp, password: value });
+      setPasswordError(false);
     }
   };
 
-  function submitForm() {
-    console.log("Submitting");
-    const resp = validateUser();
+  // does a GET /users call with temp's saved credentials
+  async function getUser(emailID, passwordID) {
+    try {
+      const resp = await axios.get("http://127.0.0.1:5000/users", {
+        auth: {
+          username: emailID,
+          password: passwordID,
+        },
+      });
 
-    if (resp === 0) {
+      // Query went through successfully
+      return resp;
```

Comment:
rather than just returning resp, I would return:
{
     success: resp.status === 200,
     user: resp.data
}

and on error....
{
   success: false,
   err: e
}